### PR TITLE
Add CODEOWNERS for Danish, Greek, Spanish, French, and Italian

### PR DESCRIPTION
**Pull request by mgeisler**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Translations
 po/da.po @mgeisler
 po/de.po @ronaldfw @fechu
+po/el.po @Mandragorian
+po/it.po @detro
```
Comment (mgeisler) [inline .github/CODEOWNERS:5]:
```suggestion
po/it.po @detro @nicomazz
```

Comment (mgeisler) [reply .github/CODEOWNERS:5]: I talked with @nicomazz and he would be happy to help out here.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Translations
 po/da.po @mgeisler
 po/de.po @ronaldfw @fechu
+po/el.po @Mandragorian
+po/it.po @detro @nicomazz
```
Comment (mgeisler) [inline .github/CODEOWNERS:5]:
```suggestion
po/fr.po @sakex
po/it.po @detro @nicomazz
```

Comment (mgeisler) [reply .github/CODEOWNERS:5]: Thanks @sakex!

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # Translations
 po/da.po @mgeisler
 po/de.po @ronaldfw @fechu
+po/el.po @Mandragorian
+po/fr.po @sakex
```
Comment (mgeisler) [inline .github/CODEOWNERS:5]:
```suggestion
po/fr.po @sakex @lb034582341
```

Thanks @lb034582341!

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,5 +1,8 @@
 # Translations
 po/da.po @mgeisler
 po/de.po @ronaldfw @fechu
+po/el.po @Mandragorian
+po/fr.po @sakex @lb034582341
+po/it.po @detro @nicomazz
```
Comment (mgeisler) [inline .github/CODEOWNERS:6]:
```suggestion
po/it.po @detro @nicomazz @bznein
```

Thanks @bznein!

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -1,5 +1,8 @@
 # Translations
 po/da.po @mgeisler
 po/de.po @ronaldfw @fechu
+po/el.po @Mandragorian
```
Comment (mgeisler) [inline .github/CODEOWNERS:4]:
```suggestion
po/el.po @Mandragorian
po/es.po @deavid
```

Comment (mgeisler) [reply .github/CODEOWNERS:4]: Thanks @deavid!

**Review by detro — APPROVED** [review]

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -1,5 +1,9 @@
 # Translations
 po/da.po @mgeisler
```
Comment (mgeisler) [inline .github/CODEOWNERS:2]:
```suggestion
po/da.po @mgeisler @voss
```

Thanks @voss!

**Pull request merged**